openapi.yml

```diff
@@ -57,6 +57,26 @@ paths:
           required: false
           schema:
             type: integer
+        - name: cocina_model_json
+          in: query
```

Comment:
Are the params provided in query string or request body?

Reply:
in this case, they are provided in the query string.  i noticed when working on this PR that the cocina JSON is given in the request body for other API endpoints.  happy to make that change, i don't feel strongly about putting it in the query string, i just hadn't realized there was a slightly different existing pattern when working on #769 (and that didn't get called out in review of that one).

Reply:
ok, changing after chatting w/ jcoyne a bit, as he pointed out that query size limits apply even though this is a POST...

Reply:
ok, just pushed an update with all the `cocina_model` related params moved to the request body